### ISSUE TITLE
DB error when starting RC

### ISSUE THREAD

**Issue by thueske** [roundcube/roundcubemail-docker#28]
Hey,

the following error occurs when I start the container. The volume already exists.

```
app_1       | Creating database schema... ERROR: [1050] Table 'session' already exists (SQL Query: CREATE TABLE `session` (
app_1       |  `sess_id` varchar(128) NOT NULL,
app_1       |  `changed` datetime NOT NULL DEFAULT '1000-01-01 00:00:00',
app_1       |  `ip` varchar(40) NOT NULL,
app_1       |  `vars` mediumtext NOT NULL,
app_1       |  PRIMARY KEY(`sess_id`),
app_1       |  INDEX `changed_index` (`changed`)
app_1       | ) /*!40000 ENGINE=INNODB */ /*!40101 CHARACTER SET utf8 COLLATE utf8_general_ci */)Error 500: [1050] Table 'session' already exists
app_1       | [FAILED]
```

Tobias

**Comment by madmath03** [comment]
Same error here.
Occurred with a postgres 10 DB (do not know if it is relevant).

**Comment by thomascube** [comment]
I'm unable to reproduce this problem.

**Comment by kiuka** [comment]
@thomascube it happens when the db already exists. would be nice to have an option for disable db install, or modify the install script in order to check if the table already exists:)

**Comment by thomascube** [comment]
This has been resolved in commit 2672a50
